cpu/efm32/periph/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 INRIA
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_MODULE_PERIPH_TIMER
+    bool "Configure timer peripheral"
+    depends on MODULE_PERIPH_TIMER
```

Comment:
I would add a dependency to `CPU_COMMON_EFM32`. Also, do all families support the LETIMER?